src/sdk.ts

```diff
@@ -1352,6 +1363,75 @@ export default class SDK {
     return this.messenger.unlisten(this.identifier, handle);
   }
 
+  /**
+   * Transaction System
+   */
+
+  /**
+   * Calls for a list of products containing a sku, displayName, and cost.
+   *
+   * @async
+   * @since 2.4.0
+   *
+   * @throws {SDKError} Will throw an error if the MuxySDK didn't load.
+   * 
+   * @return {Product[]} An array of {@link Product} 
+   * 
+   * @example
+   * const productList = sdk.getProducts();
+   */
+   public products() {
+    if (!mxy.didLoad) {
+      throw new Error('sdk.loaded() was not complete. Please call this method only after the promise has resolved.');
+    }
+
+    return this.transaction.getProducts();
```

Comment:
Since it returns a Promise, this function needs to be listed as `async` and its return type set and documented correctly.